Header header

float64 Nat_server_systemLatencyMillisec
float64 Nat_server_clientLatencyMillisec
float64 Nat_server_transitLatencyMillisec

float64 Nat_server_timeStamp
string Nat_server_timeCode

int32 Nat_server_frameID

markers markers
bodies bodies
skeletons skeletons
devices devices
devices forcePlates